assets/chezmoi.io/docs/user-guide/frequently-asked-questions/encryption.md

```diff
@@ -15,13 +15,7 @@ This can be achieved with the following process:
 First, change to chezmoi's source directory:
```

Comment:
Can you change the word `source` to either `home` or `root` so that it’s clearer that we’re talking about `.chezmoiroot`?

Reply:
Sure, but first a note about `chezmoi cd ~`. I noticed that, in my case, the shell expands the tilde to my home directory. Does it need any escaping or does any other alternative name with less collision chance exist?

Reply:
```suggestion
First, change to chezmoi's root directory:
```

Reply:
The answer to that is a little subtle. For at least 99.44% of all users, the answer is no. `chezmoi cd ~` and `chezmoi cd $HOME` and `chezmoi cd /Users/austin`would all be the same result for me, because `$HOME` is my chezmoi target directory, which is the root directory that `.chezmoiroot` maps to.

> As a side example, if I’m in `~/.config`, then doing `chezmoi cd .vim` will result in an error, `chezmoi: .vim: not in source state`. However, `chezmoi cd fish` *does* work, because `~/.config/fish` is present as `~/.local/share/chezmoi/home/private_dot_config/fish`. When additional arguments were added to `chezmoi cd`, they were added in ways that make it *easy* to get the right thing done regardless.

For remaining users, they’re explicitly doing something unusual and the place to document this particular exception is where a different target is documented.

This same principle is why I’m suggesting this file be minimally updated and that, if further work is needed, the documentation for `.chezmoiroot` be updated to indicate what portions of chezmoi’s operations would be updated.